record_mode: 0 # 0 for path tracking, 1 for virtual platoon test